File: open_tutorials_python/8 logical/2.rb
puts("아이디를 입력해주세요.\n")
input_id = gets.chomp()
puts("비밀번호를 입력해주세요.\n")
input_pwd = gets.chomp()
real_id = "geeneve"
real_pwd = "dlehdrms"

if real_id == input_id
  if real_pwd == input_pwd
    puts("Hello!")
  else
    puts("잘못된 비밀번호 입니다.")
  end
else
  puts("사용자 정보가 없습니다.")
end

File: open_tutorials_python/8 logical/2.py
input_id = input("아이디를 입력해주세요.\n")
input_pwd = input("비밀번호를 입력해주세요.\n")
real_id = "geeneve"
real_pwd = "dlehdrms"

if real_id == input_id:
    if real_pwd == input_pwd:
        print("Hello!")
    else:
        print("잘못된 비밀번호 입니다.")
else:
    print("사용자 정보가 없습니다.")
